File: BeerExercice/Clement/WikiBeer.ConsoleApp/WikiBeer.ConsoleApp/Beers/Ingredients/Cereal.cs
namespace WikiBeer.ConsoleApp.Beers.Ingredients
{
    internal class Cereal : Ingredient
    {
        public float EBC { get; private set; }

        public Cereal(string name, float ebc) : base(name, IngredientType.Cereal)
        {
            EBC = ebc;
        }
    }
}

File: WikiBeer/Web/src/models/ingredients/cereal.ts
import { Ingredient } from "./ingredient";

export class Cereal extends Ingredient{

  private _ebc: number = 0;

  constructor(id: string, name: string, description: string, ebc: number)
  {
    super(id, name, description);
    this._ebc = ebc;
  }

  get ebc(): number {return this._ebc;}
  set ebc(value: number) {this._ebc = value;}
}
